Automattic/wp-calypso
# Add Header and Footer Navbars to the Logged out plugins Page

#### Proposed Changes

More info: pau2Xa-4j7-p2

This shows the Header and Footer on the Plugins pages in the logged out view

#### Testing Instructions

* Log out
* Navigate to /plugins
* Ensure that you see the sitewide header and footer Nav bars.
* The page should be unaffected if logged in
